src/aae_exchange.erl

```diff
@@ -548,7 +540,7 @@ clock_compare(timeout, State) ->
 
 waiting_all_results({reply, not_supported, Colour}, State) ->
     aae_util:log("EX010",
-                    [Colour, State#state.exchange_id],
+                    [Colour, State#state.exchange_id, State#state.purpose],
```

Comment:
Why is this log not more similar to Line 532 in which the exchange_id comes first?